common/mount.py

```diff
@@ -604,11 +604,11 @@ def _umount(self):
         try:
             subprocess.check_call(['fusermount', '-u', self.currentMountpoint])
 
-        except subprocess.CalledProcessError:
+        except subprocess.CalledProcessError as e:
```

Comment:
One letter variable names will violate other rules.
I would suggest `err` or `exc`.

Reply:
Makes sense. I renamed it in mount.py and in tools.py